gubernator/main.py

```diff
@@ -112,24 +112,32 @@ def gcs_ls(path):
       path += '/'
     return list(gcs.listbucket(path, delimiter='/'))
 
+def get_pod_name(text):
+    """Find the pod name from the failure and return the pod name."""
+    p = re.search(r'(.*) pod (.*?) .*', text)
```

Comment:
I think it would be a good idea to put all the regexes in one place (file). Additionally, each regex should have a comment that includes:
- The file that it matches against (e.g. junit.xml, kubelet.log, etc. -- this could be a section header)
- The purpose of the regex
- An example line that it is expected to match

The goal of organizing the code this way is that if the format of the logs changes in a way that breaks some of the matchers, then it's much easier to identify which ones need to be updated.